### ISSUE TITLE
Meta: random bug triage assignments

### ISSUE THREAD

**Issue by msullivan** [python/mypy#4849]
We have a ton of bug backlog that is just sitting around without ever getting a fresh look.

One strategy might be to randomly assign everyone on the core team 5 or 10 bugs a week to take a look at, and:
 * Make sure it is triaged reasonably
 * Check to see whether it is still a bug
 * Comment about what would be needed for a fix, if they know
 * If a fix is easy or they have spare cycles, maybe fix it!

The rust team did something similar when I was an intern there, and I was a fan

**Comment by ethanhs** [comment]
I think this could be useful. Several months ago I went through about 30 issues and tested them. I think I found one or two issues that were no longer problems.

My only concern is that all of the core team is already highly pressed for time.

**Comment by ilevkivskyi** [comment]
I did this myself few times ago (I mean going through really old bugs and checking if they are still not fixed). This indeed helped to close quite a few issues (and duplicates). But last months I didn't have time for this. By the way this may be a good task for mypy sprints at PyCon. Last year the topic was usability issues, we can of course continue this year, or choose another topic -- closing old issues :-)

**Comment by ilevkivskyi** [comment]
@msullivan Another (somewhat related) idea is about tests currently skipped in mypy. I remember some time ago that few tests with `-skip` now actually work. I think it makes sense to check them and enable those that work now.

**Comment by JukkaL** [comment]
This could be useful. There are probably some open bugs that would be quick to fix for somebody with the right context. Implementing this wouldn't too hard either: a cron job that does a GitHub API call and sends a few emails might almost be sufficient. We'd probably also want to avoid sending repeated emails about the same issues. For example, we could email about a particular issue at most once every 6 months.

Some analysis: If we'd send 5 bugs to 5 people every week, we'd cover about 1300 issues a year (probably less since people won't always be available).

@ilevkivskyi Going through skipped tests is a good idea. It's probably fine to do this only once in a while.

**Comment by hauntsaninja** [comment]
This never happened; if there's time and appetite for something like this, we can open a new issue
